src/templates/list-table-block/index.vue

```diff
@@ -311,6 +321,14 @@
     isTabs: {
       type: Boolean,
       default: false
+    },
+    showSelectionMode: {
+      type: Boolean,
+      default: false
+    },
+    cleanSelectData: {
+      type: Boolean,
+      default: false
```

Comment:
Default de booleano é sempre "false"
 Sugestão de escrita simplificada:
```suggestion
    cleanSelectData: Boolean
```

Reply:
Prefiro manter o padrão com `type: Boolean` que estamos acostumados a utilizar e, além disso, essa abordagem mantém a consistência com outras propriedades semelhantes.